docs/operations/images.md

```diff
@@ -1,167 +1,235 @@
-by `kope.io`, (which is a well-known alias to account `383156758163`), with the name
-`k8s-1.4-debian-jessie-amd64-hvm-ebs-2016-10-21`.  By using a name instead of an AMI, we can reference an image
-irrespective of the region in which it is located.
+```bash
+aws ec2 describe-images --region us-east-1 --output table \
+  --owners 379101102735 \
+  --query "sort_by(Images, &CreationDate)[*].[CreationDate,Name,ImageId]" \
+  --filters "Name=name,Values=debian-stretch-hvm-x86_64-gp2-*"
+```
 
-kops should also now work on stock Debian 9 (Stretch) images.  Stock Debian 8 (Jessie) images are not recommended,
-as they typically do not have a suitable kernel and kernel options configured.
+## Debian 10 (Buster)
 
-## Ubuntu
+Debian 10 support was introduced in **Kops 1.13** and is fully functional as long as `iptables` is in `legacy` mode.
 
-Ubuntu is not the default platform, but is believed to be entirely functional.
+It is based on Kernel version **4.19** which fixes some of the bugs present in Debian 9 and effects are less visible.
 
-Ubuntu 16.04 or later is required (we require systemd).
+One notable change is the addition of `iptables` NFT, which is by default. This is not yet supported by most CNI plugins and also seems to be [slower](https://youtu.be/KHMnC3kj3Js?t=771) than the legacy version. It is recommended to switch to `iptables` legacy by using the following script in `additionalUserData` for each instance group:
 
-For example, to use Ubuntu 16.04, you could specify:
+```yaml
+additionalUserData:
+  - name: busterfix.sh
+    type: text/x-shellscript
+    content: |
+      #!/bin/sh
+      update-alternatives --set iptables /usr/sbin/iptables-legacy
+      update-alternatives --set ip6tables /usr/sbin/ip6tables-legacy
+      update-alternatives --set arptables /usr/sbin/arptables-legacy
+      update-alternatives --set ebtables /usr/sbin/ebtables-legacy
+```
 
-`image: 099720109477/ubuntu/images/hvm-ssd/ubuntu-xenial-16.04-amd64-server-20180405`
+Available images can be listed using:
 
-You can find the name for an image by first consulting [Ubuntu's image finder](https://cloud-images.ubuntu.com/locator/),
-and then using e.g. `aws ec2 describe-images --image-id ami-493f2f29`
+```bash
+aws ec2 describe-images --region us-east-1 --output table \
+  --owners 136693071363 \
+  --query "sort_by(Images, &CreationDate)[*].[CreationDate,Name,ImageId]" \
+  --filters "Name=name,Values=debian-10-amd64-*"
+```
 
-## CentOS
+## Ubuntu 18.04 (Bionic)
 
-CentOS7 support is still experimental, but should work. Please report any issues.
+Ubuntu 18.04 is fully supported and functional.
 
-The following steps are known:
+It is based on Kernel version **4.15** which has number of known bugs that affect it and may be noticed with larger clusters:
 
-* You must accept the agreement at http://aws.amazon.com/marketplace/pp?sku=aw0evgkw8e5c1q413zgy5pjce
```

Comment:
should we keep this mentioned somewhere?

Reply:
You are right. Added back in CentOS 7 section.